src/xunit.performance.api/XunitRunner.cs

```diff
@@ -4,108 +4,95 @@
 
 using System;
 using System.Threading;
-using System.Reflection;
 using Xunit.Runners;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
 
 namespace Microsoft.Xunit.Performance.Api
 {
-    public class XunitRunner
+    public static class XunitRunner
     {
-        // We use consoleLock because messages can arrive in parallel, so we want to make sure we get
-        // consistent console output.
-        private object _consoleLock = new object();
-
-        // Use an event to know when we're done
-        private ManualResetEvent _finished = new ManualResetEvent(false);
-
-        // Start out assuming success; we'll set this to 1 if we get a failed test
-        private int _result = 0;
-
-        public int Run(string assemblyPath)
+        public static int Run(string assemblyPath, string typeName = null)
```

Comment:
Thinking about this a little bit further - what's the expectation on the API user?  I think that the user should not have to plumb the typename all the way through to this API.  It's not that it's a lot of work - it's just that it is more work that the user won't know they have to do.  What I'd like to see us do is capture the typename through the constructor of the harness, which take args, and then just filter calls to RunBenchmarks based on that (xunit can do this work).  That way, we're building features like this into the system that just light up rather than making changes that require changes to end-user code.  What do you think?